staging/src/k8s.io/client-go/tools/clientcmd/api/types.go

```diff
@@ -82,6 +82,13 @@ type Cluster struct {
 	// CertificateAuthorityData contains PEM-encoded certificate authority certificates. Overrides CertificateAuthority
 	// +optional
 	CertificateAuthorityData []byte `json:"certificate-authority-data,omitempty"`
+	// ProxyURL is the URL to the proxy to be used for all requests made by this
+	// client. URLs with "http", "https", and "socks5" schemes are supported.
+	// The scheme is defaulted to "http" if no shceme is provided in the URL.
+	// This configuration takes priority of http_proxy and https_proxy
```

Comment:
```suggestion
	// This configuration takes priority over http_proxy and https_proxy
```

Reply:
with the current impl, `proxyURL: ""` does not remove a proxy if http_proxy/https_proxy envvars are set... does that match expectations? if so, it might be worth mentioning in the doc

Reply:
same typo/comments on v1/types.go

Reply:
Added a comment about "" not doing anything. This seems like reasonable behavior as long as it's documented.

Reply:
Added a comment about "" not doing anything. This seems like reasonable behavior as long as it's documented.